File: 389.find-the-difference.java
/*
 * @lc app=leetcode id=389 lang=java
 *
 * [389] Find the Difference
 */

// @lc code=start
class Solution {
    public char findTheDifference(String s, String t) {
        char result = 0;
        for (char c : s.toCharArray()) {
            result ^= c; 
        }
        for (char c : t.toCharArray()) {
            result ^= c;   
        }
        return result;
    }
}
// @lc code=end

File: 389.find-the-difference.py
#
# @lc app=leetcode id=389 lang=python
#
# [389] Find the Difference
#

# @lc code=start
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        result = 0
        for i in s:
            result^=ord(i)
        for j in t:
            result^=ord(j)
        return chr(result)
        
# @lc code=end

